doc/appendix-de-kassensichv/data-structures/data-structures.md

```diff
@@ -23,55 +23,55 @@ If you need to provide customer data in your request, you can send it in via the
+| `CustomerName` | `string (50)`  | Optional | **Name of beneficiary customer.** <br />Send via `cbCustomer` in JSON format by adding the key value pair `CustomerName` e.g. `"cbCustomer":"{"CustomerName":"Erika Musterfrau",...}"` | 1.3 |
+| `CustomerId` | `string (50)`  | Optional | **ID of the beneficiary customer.** <br />Send via `cbCustomer` in JSON format by adding the key value pair `CustomerId ` e.g. `"cbCustomer":"{"customerName":"Max Mustermann", "CustomerId":"PX9819822", ...}"` | 1.3 |
+| `CustomerType` | `string (50)`  | Optional | **Type of the beneficiary customer** (e.g. employee). <br />Send via `cbCustomer` in JSON format by adding the key value pair `CustomerType` e.g. `"cbCustomer":"{..., "CustomerId":"PX9819822", "CustomerType":"Mitarbeiter", ...}"` | 1.3 |
+| `CustomerStreet` | `string (60)`  | Optional | S**treet and house number of the beneficiary customer.** <br />Send via `cbCustomer` in JSON format by adding the key value pair `CustomerStreet` e.g. `"cbCustomer":"{..., "CustomerStreet":"Lindwurmstr. 98", ...}"` | 1.3 |
+| `CustomerZip` | `string (10)`  | Optional | **Zip of the beneficiary customer.** <br />Send via `cbCustomer` in JSON format by adding the key value pair `CustomerZip` e.g. `"cbCustomer":"{..., "CustomerZip":"80337", ...}"` | 1.3 |
+| `CustomerCity` | `string (62)`  | Optional | **City of the beneficiary customer**. <br />Send via `cbCustomer` in JSON format by adding the key value pair `CustomerCity` e.g. `"cbCustomer":"{..., "CustomerCity":"München", ...}"` | 1.3 |
+| `CustomerCountry` | `ISO 3166 ALPHA-3 country code` | Optional | **Country of the beneficiary customer.** <br />Send via `cbCustomer` in JSON format by adding the key value pair `CustomerCountry` e.g. `"cbCustomer":"{..., "CustomerCountry":"DEU", ...}"` | 1.3 |
+| `CustomerVATId` | `string(15)` | Optional | **VAT-ID of the beneficiary customer.**<br />Send via `cbCustomer` in JSON format by adding the key value pair `CustomerVATId` e.g. `"cbCustomer":"{..., "CustomerVATId":"DE123456789", ...}"` | 1.3 |
 
 
 #### Receipt case data `ftReceiptCaseData`
 
-In the general description the field `ftReceiptCaseData` is described as optional. However for the German market the content of this field is not always optional.
+In the general description, the field `ftReceiptCaseData` is described as optional. However, for the German market, the content of this field is not always optional.
 
-For some special cases we need you to transmit data within the field `ftReceiptCaseData` that is later needed for the DSFinV-K export. The following table describes when and how you have to fill them.
+For some cases, we need you to transmit data within the field `ftReceiptCaseData` that is required later for the DSFinV-K export. The following table describes when and how you have to fill them.
```

Comment:
bit better if the instructions are written with passive:
For some cases, it is needed to transmit data within the field ftReceiptCaseData, that is required later for the DSFinV-K export. The following table describes how and when these need to be filled.

Reply:
jepp